addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -307,6 +307,15 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
         font-size: 2.5rem;
     }
 
+    a.o_survey_session_navigation_next {
```

Comment:
There is already a rule below with "&.o_survey_session_navigation_next" under o_survey_session_navigation isn't it valid ?